modules/client-configuration/pages/clients-rh-cdn.adoc

```diff
@@ -200,6 +197,11 @@ The channels you need for this procedure are:
 | -
 | res8-manager-tools-pool-x86_64
 
+| {redhat} 9
+| rhel9-pool-x86_64
+| -
+| res9-suse-manager-tools-x86_64
```

Comment:
I would say this was changed, and now instead of res9 we use el9.
Please check with @juliogonzalez